carbon/db.py

```diff
@@ -101,8 +105,26 @@ def __call__(self) -> Engine:
         )
         raise AttributeError(nonconfigured_engine_error_message)
 
-    def configure(self, conn: str) -> None:
-        self._engine = self._engine or create_engine(conn)
+    def configure(self, conn: str, **kwargs: Any) -> None:  # noqa: ANN401
```

Comment:
Let's start expanding some of these variable names like `conn`

Reply:
Please see the latest commit! The only line of code where I kept `conn` was in `CarbonCopyFTPS.storbinary`; the class inherits from `FTP_TLS`, which uses the `conn` naming convention.